docs/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Changes are grouped as follows:
 ## Improved
 
 - Speed of nodes & edges generation
+- Multi namespace support for nodes & edges generation (see [feature request](https://github.com/cognitedata/neat/issues/171))
```

Comment:
Earlier PR? There are no changes to the package here, only tests.

Reply:
It is a result of previous PR :)